### PR TITLE
Feature deployment to gcp

### PR DESCRIPTION
**Pull request by bradleySuira**
- Added deployment assets to Google cloud
- Fixed for comments in the last PR (removed session from slots manager and session management redundant code in dashboard)
- Change ui colors and icons for new branding 
- Fixed errors with channel processing in Slots
- Added alpha package for contracts

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  platform-frontend:
+    image: us.gcr.io/dbet-platform/platform-fronend:latest
```
Comment (decent-dev) [inline docker-compose.yml]: Shouldn't this be `us.gcr.io/dbet-platform/platform-frontend:latest`

File: package.json [hunk 2]
```diff
@@ -29,127 +29,129 @@
...
+    "solidity-sha3": "^0.4.1",
+    "swarm-js": "^0.1.37",
+    "thor-devkit": "^0.2.3",
+    "thorify": "0.3.6",
+    "truffle-hdwallet-provider": "^0.0.6",
+    "web3": "^1.0.0-beta.35",
+    "wowjs": "^1.1.3"
+  },
+  "scripts": {
+    "start": "node scripts/start.js DEBT_PLATFORM_FRONTEND",
```
Comment (decent-dev) [inline package.json]: These'd need to be `DBET_PLATFORM_FRONTEND`

File: docker-compose.yml (hunk 1 above)
Comment (bradleySuira) [reply docker-compose.yml]: fixed

File: package.json (hunk 2 above)
Comment (bradleySuira) [reply package.json]: fixed

**Comment by bradleySuira** [comment]
moved to alpha, wil request the PR from alpha to development

**Pull request closed without merging**